### DISCUSSION TITLE
질문!!

### DISCUSSION THREAD

**Discussion by Rla-oes** [lSH1234321/gyeseong_forum#8] [General]
인공지능을 개발할 때 어느 관점을 가지고 하는 게 좋을까요?
30503 김민서

**Comment by dbrud1101** [comment]
합리적으로 생각하고 인간처럼 행동하기

**Reply by dbrud1101** [reply]
굳이 인간적으로 사고하지 않아도 더 합리적인 방법이 있다면 합리적인 방식으로 사고하는 것이 결과적으로 더 효율을 추구할 수 있다고 생각한다.  그러나 인간을 대하는 경우에만 사회적으로 여러가지 요소를 고려하여 그 상황에서 더 효율적일지 않다고 하더라고 인간적인 면모를 보일 필요가 있다고 생각한다.

**Comment by Rla-oes** [comment]
rationally

**Reply by Rla-oes** [reply]
챗GPT가 질문을 이해하고 답변하는 것처럼 보이지만 사실은 그것이 아니라 단순히 질문에 대한 데이터를 모아서 답변을 하는 것처럼 인공지능이 사람처럼 생각하는 것은 불가능할 것 같다. 따라서 인공지능을 만드는 이유가 인간에게 도움을 주기 위해서라면 윤리적인 판단을 내리는 건 인간에게 맡기고 인공지능은 윤리적으로 문제가 되는 부분을 제외한 것 (합리적이고 이성적인 판단만 요구하는 것... 정해진 답이 있는 것?)을 해결하는 것이 좋을 것 같다.

**Comment by SystemMessage** [comment]
rationally..

**Reply by SystemMessage** [reply]
인공지능이란 결국 인간과 유사한 행동 혹은 판단을 내리는 인간이 만든 도구에 불과한데, 만약 인공지능이 인간처럼 사고하고 행동한다면 인간의 사고방식이 그러하듯, 굉장히 불합리하거나 편향된 결론을 내릴 수 있다. 앞에서 말했듯, 인공지능은 결국 인간의 도구로써의 기능을 효율적으로 해야하기 때문에 절대적으로 합리적인 판단만을 하는 것이 좋다고 생각한다.

**Comment by lSH1234321** [comment]
Thinking humnaly나 Acting humanly가 바람직할 것 같아요.. rationally로 행동하면 이익만을 추구하다가 인간적 가치를 상실할지도?

**Reply by lSH1234321** [reply]
합리적으로 사고하고 행동하는 것이 효율적일 수는 있으나, 인공지능이 윤리적 판단의 영역에 들어서기 시작할 때는 인간적인 사고가 필요할 것이라고 판단했다. 무조건 이성적인 건 배척해야 한다고 생각하기도 했으나, 오히려 "인간적"이라는 것이 편향을 대표하는 것이 될지도 모르겠다.
